src/main/java/trello/ui/pages/modal/CardModal.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package trello.ui.pages.modal;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import trello.ui.pages.BasePage;
+
+import java.util.List;
+
+/**
+ * CardModal class.
+ *
+ * @author Josue Rodriguez Garcia.
+ * @version 0.0.1
+ */
+public class CardModal extends BasePage {
+    @FindBy(className = "js-card-detail-title-input")
```

Comment:
Please, add a white line above this line.

Reply:
Solved